client/src/assets/messages/scorecard_summary_messages.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Text to use on the scorecard summary page.
+ */
+export class ScorecardSummaryMessages {
+    static SCORECARD_SUMMARY_PANEL_HEADER = "What's in your score";
```

Comment:
nit: single quotes to be consistent with other strings

Reply:
yeah good point